kubernetes/client/rest.py

```diff
@@ -39,10 +39,7 @@ def __init__(self, resp):
         self.reason = resp.reason
         self.data = resp.data
 
-    def getheaders(self):
-        """Returns a dictionary of the response headers."""
-        return self.urllib3_response.getheaders()
```

Comment:
This file is generated by openapi-generator. Has this method been removed in a newer version of the generator? If so, was there a replacement?

Reply:
To clarify, is it okay to simply drop this method? I'd expect that there are other places depending on this, and would be broken if we simply drop this method.

Reply:
It is used e.g. here: https://github.com/kubernetes-client/python/blob/master/kubernetes/client/exceptions.py#L91

I would rather suggest to keep and fix it. e.g.

```python
    def getheaders(self):
        """Returns a dictionary of the response headers."""
        return self.urllib3_response.headers
```